stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -561,11 +561,10 @@ public class ChannelController internal constructor(
     internal suspend fun editMessage(message: Message): Result<Message> {
```

Comment:
Can we move this method to `ChannelLogic`?

Reply:
Actually, this method isn't being used anymore. We should removed it. This logic is already in the `OfflinePlugin.onMessageEditRequest` and `OfflinePluginonMessageEditResult`

Reply:
Or deprecate it. Let me see...

Reply:
Depreated